ci/tests/vpool/vpool_test.py

```diff
@@ -91,3 +95,55 @@ def ovs_2263_verify_alba_namespace_cleanup_test():
         alba.delete_namespace(BACKEND_NAME, namespace['name'])
     result = alba.list_namespaces(BACKEND_NAME)
     assert len(result) == 0, "Expected no namespaces present on the {1} backend, found {2}".format(no_namespaces, BACKEND_NAME, len(result))
+
+
+def ovs_2703_kill_various_services_test():
+    """
+    Kill various services and see if they recover
+    %s
+    """ % general.get_function_name()
+
+    general.check_prereqs(testcase_number=3,
+                          tests_to_run=testsToRun)
+    issues_found = ''
+    api = Connection.get_connection()
+    vpool_list = api.get_component_by_name('vpools', VPOOL_NAME)
+    if not vpool_list:
+        generic.add_generic_vpool()
+
+    services_folder = '/opt/OpenvStorage/config/templates/systemd/'
+    out, err = general.execute_command('ls {0}'.format(services_folder))
+
+    services_to_kill = out.splitlines()
+    for index in range(len(services_to_kill)):
+        services_to_kill[index] = services_to_kill[index].split('.')[0]
+
+    client = SSHClient(ENV_IP, username='root')
+
+    for master_service in services_to_kill:
+        all_services, err = general.execute_command("initctl list | grep {0}".format(master_service))
+        if not err and len(all_services):
+            for service in all_services.splitlines():
+                service_name = service.split(' ')[0]
+                if ServiceManager.has_service(service_name, client) is False:
+                    issues_found += 'Service {0} not modeled even if running on system\n'.format(service_name)
+                service_status = service.split(' ')[1][:-1]
+                service_proc_id = service.split(' ')[3][:-1]
+                if service_status not in 'start/running':
+                    issues_found += 'Service {0} not found in running state\n'.format(service)
+                else:
+                    out, err = general.execute_command("kill -9 {0}".format(service_proc_id))
+                    time.sleep(5)
```

Comment:
will this always work?
perhaps we could wait for x minutes and check if a new pid appears with the process involved ...